src/components/multi-select/_multi-select.scss

```diff
@@ -13,11 +13,11 @@
 @import '../list-box/list-box';
 
 @include exports('multi-select') {
-  .bx--multi-select.bx--combo-box > .bx--list-box__field {
+  .#{$prefix}--multi-select.#{$prefix}--combo-box > .#{$prefix}--list-box__field {
```

Comment:
I'm also noticing that box-shadow here, but it looks like we never added the `gray-50` border bottom. Perhaps this is a bigger change than I thought

Reply:
Might be better in a separate issue around list box styles, there is a lot going on here 🙃

Reply:
Yeah, I'll work on that this morning and fix that up. Disregard!